guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -35,45 +35,42 @@ If you set a wildcard value `*` for the certificate's Common Name `CN =` in the
 If the command is successful, it returns two `{certs-generate}` commands, one of which you must use to generate the certificate archive file for your {SmartProxyServer}.
 ifdef::satellite[]
 +
-.Example output of `katello-certs-check`
+.To use them inside a NEW $CAPSULE, run this command:
 [options="nowrap", subs="+quotes,attributes"]
 ----
-Validation succeeded.
-
-To use them inside a NEW $CAPSULE, run this command:
-
 capsule-certs-generate --foreman-proxy-fqdn "$CAPSULE" \
     --certs-tar  "~/$CAPSULE-certs.tar" \
     --server-cert "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_" \
     --server-key "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_" \
     --server-ca-cert "_/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_" \
-
-To use them inside an EXISTING $CAPSULE, run this command INSTEAD:
-
+----
++
+.To use them inside an EXISTING $CAPSULE, run this command INSTEAD:
```

Comment:
The only difference is the additional `--certs-update-*` arguments. I'm not sure how much that is actually still needed, since we've fixed various idempotency bugs in that area.

I'd prefer to align on a single set of instructions. AFAIK the update instructions do not hurt on a new installation, so perhaps we should use that for all.

@ehelms you worked in this area. Thoughts?

Reply:
I think to be on the safe side I'd need to just test if we have changed things enough to not need the parameters. They come from the installer itself:

https://github.com/theforeman/foreman-installer/blob/develop/hooks/pre/20-certs_update.rb